content/blog/2023/2023-08-31-fairness-hiding-protected-attribute.md

```diff
@@ -0,0 +1,56 @@
++++
+author = "Žan Mervič"
+date = "2023-08-31"
```

Comment:
I really like this blog post. The structure is fantastic, it addresses a questions and aims to answer it. I think all your post should be structured more like this one.

Also, it addresses a very important question and should therefore be one the first blog posts. Maybe the second or third?